GPU/GLES/Framebuffer.cpp

```diff
@@ -2086,7 +2086,7 @@ bool FramebufferManager::GetDepthbuffer(u32 fb_address, int fb_stride, u32 z_add
 	if (vfb->fbo)
 		fbo_bind_for_read(vfb->fbo);
 	if (gl_extensions.GLES3 || !gl_extensions.IsGLES)
-		glReadBuffer(GL_DEPTH_ATTACHMENT);
+		glReadBuffer(GL_COLOR_ATTACHMENT0);
```

Comment:
Right, well, it does it also when getting the color buffer, so I figured the sane thing was to make them match in case we ever do use other attachments.

-[Unknown]